README.md

```diff
@@ -47,7 +47,14 @@ Getting Started
     bin/ycsb run basic -P workloads/workloada
     ```
 
-  Running the `ycsb` command without any argument will print the usage. 
+   Running the `ycsb` command without any argument will print the usage. 
+   Inorder to use the multiGet feature where a set of 'gets' can be batched and sent to the server in one
```

Comment:
please call this out as an optimization. Also note that it is effectively buffering the get calls client side, since that impact comparison with other systems.